plugins/woocommerce/src/Internal/Admin/ProductReviews/Reviews.php

```diff
@@ -511,6 +511,8 @@ protected function get_pending_count_bubble() : string {
 			]
 		);
 
+		$count = apply_filters( 'woocommerce_product_reviews_pending_count', $count );
```

Comment:
* We should add a docblock here.
* It could also be valuable to provide the `$args` (which we would break out from the earlier call to `get_comments()` as a second parameter.

Reply:
Added  a docblock here.